test/CMakeLists.txt

```diff
@@ -63,9 +63,13 @@ foreach (algo_index RANGE 28) # 42
 endforeach ()
 
 if (Python_FOUND AND NUMPY_FOUND AND (SWIG_FOUND OR (EXISTS ${PROJECT_SOURCE_DIR}/src/swig/nlopt-python.cpp)))
-  set (PYINSTALLCHECK_ENVIRONMENT "LD_LIBRARY_PATH=${PROJECT_BINARY_DIR}/src/swig"
-                                  "PYTHONPATH=${PROJECT_BINARY_DIR}/src/swig"
-    )
+  if (CMAKE_HOST_SYSTEM_NAME MATCHES Windows)
+    set (PYINSTALLCHECK_ENVIRONMENT "PYTHONPATH=${PROJECT_BINARY_DIR}/src/swig/python;${PROJECT_BINARY_DIR}/src/swig")
+  else()
+    set (PYINSTALLCHECK_ENVIRONMENT "LD_LIBRARY_PATH=${PROJECT_BINARY_DIR}/src/swig"
+                                    "PYTHONPATH=${PROJECT_BINARY_DIR}/src/swig/python:${PROJECT_BINARY_DIR}/src/swig"
```

Comment:
it does not hurt to set LD_LIBRARY_PATH on windows, that avoids the unnecessary if/else

Reply:
Actually, since you bumped the minimum CMake version in master from 3.13 to 3.15, we can now simply use https://cmake.org/cmake/help/latest/manual/cmake-generator-expressions.7.html#genex:SHELL_PATH with semicolon-separated paths to let CMake automatically pick the proper separator, a feature that was introduced in CMake 3.14.

Reply:
Done in the latest revision (commit 427ca219a7ebd670513b8f0a4229facddb7dd229).